app/domain/logs.rb

```diff
@@ -188,6 +188,20 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnGCP
+
+      ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracting resource restrictions from role '{0-role-id}' with prefix '{1-prefix}'",
```

Comment:
@shulifink please review

Reply:
Fine - see comment in line 200 about caps